runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -13,6 +13,7 @@
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
+import java.util.concurrent.TimeoutException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.concurrent.TimeoutException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)